fe/fe-core/src/main/java/org/apache/doris/datasource/CatalogMgr.java

```diff
@@ -462,6 +464,20 @@ public CatalogIf replayCreateCatalog(CatalogLog log, boolean isReplay) throws Dd
         writeLock();
         try {
             CatalogIf catalog = CatalogFactory.constructorFromLog(log);
+            Map<String, String> props = log.getProps();
+            if (props.containsKey(METADATA_REFRESH_INTERVAL_SEC)) {
+                // need refresh
+                String catalogName = log.getCatalogName();
+                Integer metadataRefreshIntervalSec = null;
+                try {
+                    metadataRefreshIntervalSec = Integer.valueOf(props.get(METADATA_REFRESH_INTERVAL_SEC));
+                } catch (NumberFormatException e) {
+                    throw new DdlException("Invalid properties: " + METADATA_REFRESH_INTERVAL_SEC);
+                }
+                Integer[] sec = {metadataRefreshIntervalSec, metadataRefreshIntervalSec};
+                Env.getCurrentEnv().getRefreshManager().registerIntoRefreshMap(catalogName, sec);
```

Comment:
You also need to modify the method `gsonPostProcess()` in this class.
To add all catalogs which need to refresh to the RefreshManger, or these refresh info will be lost.

You can test this bug with following steps:
1. create a catalog with refresh property.
2. Restart FE, and wait for 2 minutes.
3. Restart FE again, you will see that the catalog won't refresh anymore.